src/Kernel.php

```diff
@@ -15,55 +18,19 @@
  * limitations under the License.
  */
 
-namespace App;
+namespace Surfnet\StepupSelfService;
 
 use Symfony\Bundle\FrameworkBundle\Kernel\MicroKernelTrait;
-use Symfony\Component\Config\Loader\LoaderInterface;
-use Symfony\Component\Config\Resource\FileResource;
-use Symfony\Component\DependencyInjection\ContainerBuilder;
 use Symfony\Component\HttpKernel\Kernel as BaseKernel;
-use Symfony\Component\Routing\RouteCollectionBuilder;
+use function dirname;
```

Comment:
Our (unwritten) rule is to not import functions. I'm not super mad about this, and if you prefer to leave as is, I can certainly live with it. It is a marginal speed gain (albeit very marginal).